spec/controllers/users_controller_spec.rb

```diff
@@ -30,10 +43,24 @@
     end
   end
 
+  context "GET #show" do
+    context "when not logged in" do
+      it "won't access the show page" do
+        user = create(:user)
+
+        get :show, params: { id: user.id }
+
+        expect(flash[:alert]).
+          to match I18n.t("sessions.new.logged_in")
+      end
+    end
+  end
+
   context "GET #edit" do
     context "when invalid parameters" do
       it "won't update a user's profile" do
         user = create(:user, name: "John")
+        log_in_as user
```

Comment:
I would stub the controller's `current_user` method here

Reply:
Done in sha: 5e2fd92